### PR TITLE
Add ability to group media by stacking for visualization of tracked media progression

### PR DESCRIPTION
**Pull request by KaroMourad**
Add: 
 
 - Group stacking button in Media Controls
 - Group images by stacking by last grouped value per media list
 - Slider to select stacked images 
 - Input dropdown to select stacked images

### CODE REVIEW

**Review by rubenaprikyan — APPROVED** [review]
lgtm !

**Review by rubenaprikyan — APPROVED** [review]

**Comment by KaroMourad** [comment]
Resolves #1225

**Review by roubkar — APPROVED** [review]

File: aim/web/ui/src/components/MediaList/ImageBox.tsx [hunk 1]
```diff
@@ -71,6 +71,7 @@ const ImageBox = ({
     e.stopPropagation();
     setIsImageFullViewPopupOpened(true);
   }
+
   // TODO need to add focused image logic
   // function safeSyncHoverState(args: any): void {
```
Comment (Hamik25) [inline aim/web/ui/src/components/MediaList/ImageBox.tsx:76]: Please remove comments.

File: aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx [hunk 2]
```diff
@@ -103,6 +105,27 @@ function Controls(
...
+        title={props.additionalProperties.stacking ? 'Group stacked' : 'Stack'}
+      >
+        <div
+          className={classNames('Controls__anchor', {
+            active: props.additionalProperties.stacking,
+            outlined: props.additionalProperties.stacking,
+            disabled: _.isEmpty(props.orderedMap),
+          })}
+          onClick={
+            !_.isEmpty(props.orderedMap) ? props.onStackingToggle : undefined
```
Comment (Hamik25) [inline aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx:118]: Here you will receive the react warning about the synthetic events please replace undefined to noop or you can have conditional props which can combine onClick prop conditionally.

File: aim/web/ui/src/components/MediaList/ImageBox.tsx (hunk 1 above)
Comment (KaroMourad) [reply aim/web/ui/src/components/MediaList/ImageBox.tsx:76]: As we didn't add this functionality I will remove it, later if we need it i will add, thanks

File: aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx (hunk 2 above)
Comment (KaroMourad) [reply aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx:118]: i try reproduce warnings but i can't , but for safety and readability i will change it to noop, thanks

File: aim/web/ui/src/components/MediaList/ImageBox.tsx (hunk 1 above)
Comment (KaroMourad) [reply aim/web/ui/src/components/MediaList/ImageBox.tsx:76]: will remove, it's unnecessary for now , thanks

File: aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx (hunk 2 above)
Comment (KaroMourad) [reply aim/web/ui/src/pages/ImagesExplore/components/Controls/Controls.tsx:118]: tried to reproduce warnings but I can't, but for safety and readability I will change it to `noop`, thanks

**Pull request merged**